ws-nextjs-app/pages/_document.page.tsx

```diff
@@ -140,6 +159,19 @@ export default class AppDocument extends Document<DocProps> {
     const helmetScriptTags = helmet.script.toComponent();
 
     switch (true) {
+      case isAmp && pageType === 'article':
```

Comment:
Is this more of a localhost measure? In reality, the upstream routing system will only have "valid" `.amp` routes defined, so nothing will get through that we don't want.

Reply:
No worries, just something that we'll need to maintain and extend the more page types we move over I guess?

Agree that it may be confusing for devs if they can access a page on AMP locally but not on Live.